src/Core/functions.php

```diff
@@ -325,3 +326,108 @@ function deprecationWarning(string $version, string $message, int $stackFrame =
         trigger_error($message, E_USER_DEPRECATED);
     }
 }
+
+/**
+ * Converts the given value to a string.
+ *
+ * This method attempts to convert the given value to a string.
+ * If the value is already a string, it returns the value as it is.
+ * If the conversion is not possible, it returns NULL.
+ *
+ * @param mixed $value The value to be converted.
+ * @return ?string Returns the string representation of the value, or null if the value is not a string.
+ * @since 5.1.0
+ */
+function toString(mixed $value): ?string
```

Comment:
This docblock needs to describe the nuances of each conversion such as bool to a 1 and 0 including failure cases.

Reply:
I can rewrite the docs when this is merged.